.travis.yml

```diff
@@ -5,7 +5,7 @@ os:
   - osx
 
 go:
-- "1.11"
+- "1.12"
```

Comment:
we should ensure the version check we have is also bumped up to 1.12.

Reply:
nah, it's just for building KB, I believe.  Everything else works fine with Go 1.11 (in fact, building KB also would work fine, except that Travis's Go 1.11 is slightly old and has a bug).